applications/DEMApplication/python_scripts/DEM_procedures.py

```diff
@@ -760,6 +760,12 @@ def RemoveFoldersWithResults(self, main_path, problem_name, run_code=''):
         except OSError:
             pass
 
+        try:
+            file_to_remove = os.path.join(main_path, 'flux_data_new.hdf5')
```

Comment:
@farrufat-cimne  This file is produced by very specific runs and can be deleted by the specific cases.

Reply:
fixed